src/main/java/seedu/address/model/room/RoomList.java

```diff
@@ -106,20 +110,35 @@ public void addRooms(int numOfRooms) {
     public void addRooms(Room room) {
         this.numOfRooms++;
         rooms.add(room);
-        roomObservableList.add(room);
+        internalList.add(room);
     }
+
     /**
-     * Sets the elements of {@code roomObservableList}.
+     * Adds a task to a room.
+     *
+     * The room must exist in the {@code RoomList}.
      *
-     * @param roomList PriorityQueue containing all the rooms.
+     * @param task The task to add.
+     * @param room The room to which the task should be added.
+     * @throws RoomNotFoundException if {@code room} is not in {@code RoomList}.
      */
-    private void convertPriorityQueue(PriorityQueue<Room> roomList) {
```

Comment:
thanks for cleaning up my unused method from before! 👍

Reply:
No problem!